### PR TITLE
git-sizer 1.0.0 (new formula)

### PR DESCRIPTION
**Pull request by sgerrand**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

I've sorted out the issues with #24912 and this formula can be both built from source and HEAD.

### CODE REVIEW

**Comment by sgerrand** [comment]
👋 @fxcoudert: For your attention.

**Comment by sgerrand** [comment]
The output from running `git-sizer` on this branch:
```
git-sizer
Processing blobs: 127609
Processing trees: 218122
Processing commits: 109048
Matching commits to trees: 109048
Processing annotated tags: 0
Processing references: 20
| Name                         | Value     | Level of concern               |
| ---------------------------- | --------- | ------------------------------ |
| Overall repository size      |           |                                |
| * Trees                      |           |                                |
|   * Total size               |  12.9 GiB | ******                         |
|   * Total tree entries       |   361 M   | *******                        |
|                              |           |                                |
| Biggest objects              |           |                                |
| * Trees                      |           |                                |
|   * Maximum entries      [1] |  4.49 k   | *                              |

[1]  a30014335ba09665b3a53f0cd4184fc8497762e9 (37d2e8b8bd010ab9156858c49fde4f9a868854a7:Formula)
```

File: Formula/git-sizer.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  end
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/github.com/github/git-sizer"
+    dir.install buildpath.children
+
+    cd buildpath/"src" do
```
Comment (fxcoudert) [inline Formula/git-sizer.rb]: `cd "src" do` should suffice

File: Formula/git-sizer.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+class GitSizer < Formula
+  desc "Compute various size metrics for a Git repository"
+  homepage "https://github.com/github/git-sizer"
+  url "https://github.com/github/git-sizer/archive/v1.0.0.tar.gz"
+  sha256 "8ef6b2adb646ccb72f5ccfebd19a57371b119da1aac02ef06f1a46af6f7447a0"
+
+  head do
+    url "https://github.com/github/git-sizer.git"
```
Comment (fxcoudert) [inline Formula/git-sizer.rb]: Single line, `head "https://github.com/github/git-sizer.git"`, after `sha256` with no blank line in-between

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/git-sizer.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    cd buildpath/"src" do
+      system "go", "build", "-o", bin/"git-sizer", "github.com/github/git-sizer"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    system "git", "init"
+    output = shell_output("#{bin}/git-sizer")
+
```
Comment (fxcoudert) [inline Formula/git-sizer.rb]: Remove blank line

File: Formula/git-sizer.rb (hunk 2 above)
Comment (sgerrand) [reply Formula/git-sizer.rb]: :recycle: Implemented in 8725b5e4bb.

File: Formula/git-sizer.rb (hunk 1 above)
Comment (sgerrand) [reply Formula/git-sizer.rb]: :recycle: Implemented in 8725b5e4bb.

File: Formula/git-sizer.rb (hunk 3 above)
Comment (sgerrand) [reply Formula/git-sizer.rb]: :recycle: Implemented in 8725b5e4bb.

**Comment by sgerrand** [comment]
@fxcoudert: I've made the changes you requested.

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @sgerrand for the pull request!

**Pull request closed without merging**